apps/ui/src/hooks/Messages/fetchMessagesForUser.ts

```diff
@@ -25,13 +25,10 @@ const useGetUserMessages = (userId: number) => {
     [userId, 'getUserMessages'],
     () => fetchMessagesForUser(userId),
     {
-      onError: (error: any) => {
+      onError: (error: Error) => {
         console.error('Error getting userMessages', error);
         showMessage(TEXT.alerts.REQUEST_FAILED, 'error');
       },
-      onSuccess: () => {
```

Comment:
dropped this cause it was super random unless I'm missing something (this raises a notification on GET)